tf2onnx/rewriter/__init__.py

```diff
@@ -49,6 +48,7 @@
     "rewrite_quantize_and_dequantize",
     "rewrite_layer_normalization",
     "rewrite_conv_dilations",
+    "rewrite_conv2d_with_pad",
```

Comment:
`rewrite_biasadd_with_conv2d` fuses change the tf op `Conv2D` to ONNX op 'Conv'.
So `rewrite_conv2d_with_pad` should be set after `rewrite_biasadd_with_conv2d`.